egs/wsj/asr1/run.sh

```diff
@@ -21,6 +21,8 @@ seed=1
 # feature configuration
 do_delta=false
 
+# config files
+preprocess_config=conf/no_preprocess.yaml  # use conf/specaug.yaml for data augmentation
```

Comment:
Is the comment means if I change this line to preprocess_config=conf/specaug.yaml, The spec_augment module would work? If not, how can I use specaug function?

Reply:
check #830

Reply:
@jiayu2B yes